diff -Naur shermans_aquarium-3.0.0-old/shermans/status_linux24.c shermans_aquarium-3.0.0/shermans/status_linux24.c
--- shermans_aquarium-3.0.0-old/shermans/status_linux24.c	2004-08-05 20:49:18.000000000 +0200
+++ shermans_aquarium-3.0.0/shermans/status_linux24.c	2004-10-21 07:47:33.030028064 +0200
@@ -65,8 +65,7 @@
     if(sensor_type == SENSORS_FAN1 || sensor_type == SENSORS_FAN2){
 	fscanf(status_file,"%d %d\n", &status1, &status2);
 	fclose(status_file);
-	if(flag == SENSORS_CURRENT)
-	    return status2;
+    	return status2;
     }
 
     return -1;
